modules/web/sites/hybrid-connection-namespaces/relays/main.bicep

```diff
@@ -0,0 +1,58 @@
+@description('Required. The resource ID of the relay namespace hybrid connection.')
+param hybridConnectionResourceId string
+
+@description('Conditional. The name of the parent web site. Required if the template is used in a standalone deployment.')
+param appName string
+
+@description('Optional. Name of the authorization rule send key to use.')
+param sendKeyName string = 'defaultSender'
+
+@description('Optional. Location for all Resources.')
+param location string = resourceGroup().location
+
+@description('Optional. Enable telemetry via a Globally Unique Identifier (GUID).')
+param enableDefaultTelemetry bool = true
+
+var splitResourceId = split(hybridConnectionResourceId, '/')
```

Comment:
I can convert to using the parameter directly, but it seemed more efficient to split the parameter one time instead of 4 times? That was my only thought.

Reply:
You raise a good point. If this introduces efficiency, I can create another issue as a feature request to discuss and align other CARML modules to a standard pattern. May I suggest we continue to have the same implementation pattern for now?

Reply:
That sounds good to me.